### PR TITLE
new resource: `azurerm_sentinel_data_connector_azure_advanced_threat_protection`'

### PR DESCRIPTION
**Pull request by magodo**
new resource: `azurerm_sentinel_data_connector_azure_advanced_threat_protection`'

### CODE REVIEW

File: website/docs/r/sentinel_data_connector_azure_advanced_threat_protection.html.markdown [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+The following arguments are supported:
+
+* `log_analytics_workspace_id` - (Required) The ID of the Log Analytics Workspace that this Azure Advanced Threat Protection Data Connector resides in. Changing this forces a new Azure Advanced Threat Protection Data Connector to be created.
+
+* `name` - (Required) The name which should be used for this Azure Advanced Threat Protection Data Connector. Changing this forces a new Azure Advanced Threat Protection Data Connector to be created.
+
+---
+
+* `tenant_id` - (Optional) The ID of the tenant that this Azure Advanced Threat Protection Data Connector connects to. Changing this forces a new Azure Advanced Threat Protection Data Connector to be created.
```
Comment (tombuildsstuff) [inline website/docs/r/sentinel_data_connector_azure_advanced_threat_protection.html.markdown:44]:
```suggestion
* `tenant_id` - (Optional) The ID of the tenant that this Azure Advanced Threat Protection Data Connector connects to. Changing this forces a new Azure Advanced Threat Protection Data Connector to be created.

-> **Note:** If unspecified the Tenant ID of the current Subscription will be used.
```

**Review by tombuildsstuff — APPROVED** [review]
hey @magodo 

Thanks for this PR - I've taken a look through and left one comment inline but on the whole this looks pretty good, if we can fix up the comments then this should be otherwise good to go 👍 

Thanks!

File: website/docs/r/sentinel_data_connector_azure_advanced_threat_protection.html.markdown [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+subcategory: "Sentinel"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_sentinel_data_connector_azure_advanced_threat_protection"
+description: |-
+  Manages a Azure Advanced Threat Protection Data Connector.
+---
+
+# azurerm_sentinel_data_connector_azure_advanced_threat_protection
+
+Manages a Azure Advanced Threat Protection Data Connector.
```
Comment (manicminer) [inline website/docs/r/sentinel_data_connector_azure_advanced_threat_protection.html.markdown:11]:
```suggestion
Manages a Azure Advanced Threat Protection Data Connector.

!> **NOTE:** This resource requires that [Enterprise Mobility + Security E5](https://www.microsoft.com/en-us/microsoft-365/enterprise-mobility-security) is enabled on the tenant being connected to.
```

**Review by manicminer — APPROVED** [review]
Thanks for this @magodo. I've added a note to the docs about the AAD license requirements as well as `securityinsight.AATPDataConnector` to the supported types in the import validation function.

Screenshot 2021-03-11 at 01 23 34

**Pull request merged**